src/libraries/System.Net.WebSockets.Client/src/System.Net.WebSockets.Client.csproj

```diff
@@ -1,7 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <AllowUnsafeBlocks>True</AllowUnsafeBlocks>
-    <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;$(NetCoreAppCurrent)-Unix</TargetFrameworks>
+    <NoWarn>$(NoWarn);CS1573</NoWarn>
```

Comment:
Nit: seems easier for us to just update Triple-slash docs. Not sure why this PR is even hitting this.